### ISSUE TITLE
Do not jump to the latest commit each time I focus on the app

### ISSUE THREAD

**Issue by firedev** [laullon/gitx#124]
I keep Gitx open to see changes in a commit and each time I focus on the app it jumps to the latest commit. Very annoying.

**Comment by Uncommon** [comment]
Same as #102.

**Comment by mcmire** [comment]
And #91, and #87.

**Comment by mcmire** [comment]
There is an option "refresh automatically" you can uncheck that will prevent this from happening. You have to do a manual Cmd-R if you really want to refresh.

**Comment by firedev** [comment]
The thing is, I do want Stage view to be updated automatically. But I don't want the "Main" view to be constantly refreshed. I think this checkbox behaviour should be changed to not affect the Main and the Branch view.

**Comment by kingbuzzman** [comment]
So.. its not a bug. It's a feature? ... kinda annoying. But thanks for the quick fix

**Comment by Uncommon** [comment]
I'd call it a bug - changing the selection should not be a side effect of refreshing.

**Comment by firedev** [comment]
Yep, I don't mind refreshing Stage view, but when I am browsing the history and trying to pinpoint the change made in the past the last thing I want to do is to start all over again. So checkbox should affect only Stage I think.

**Comment by jphalip** [comment]
This was fixed in 3e045a4
